Sources/_CryptoExtras/AES/Block Function.swift

```diff
@@ -131,9 +131,10 @@ extension AES {
         init(_ blockBytes: BlockBytes) {
             self.blockBytes = blockBytes
         }
-
-        init(_ iv: AES._CBC.IV) {
-            self.blockBytes = iv.ivBytes
+        
+        init<BlockBytes: Sequence>(blockBytes: BlockBytes) where BlockBytes.Element == UInt8 {
+            let blockBytes: [UInt8] = Array(blockBytes)
+            self.init(blockBytes: blockBytes)
```

Comment:
I get the impression we can keep the existing constructor here, as the tuple is still present now.

Reply:
I added the original constructor back in